test/test_postman.py

```diff
@@ -48,26 +65,30 @@ def test_init_ganache():
 
 @pytest.mark.web3_deploy
 def test_deploy_l1_contracts():
-    """Deploys Ethereum contracts in the Ganache instance, including the L1L2Example and MockStarknetMessaging contracts"""
+    """Deploys Ethereum contracts in the Hardhat testnet instance, including the L1L2Example and MockStarknetMessaging contracts"""
 
-    global MESSAGING_CONTRACT_ADDRESS # pylint: disable=global-statement
```

Comment:
I think pylint disable statements can be applied to a whole file. But an even better solution would be this: https://stackoverflow.com/questions/44441929/how-to-share-global-variables-between-tests

Reply:
Changed